src/Features/Core/Portable/Completion/Providers/AbstractDocCommentCompletionProvider.cs

```diff
@@ -52,7 +52,9 @@ internal abstract class AbstractDocCommentCompletionProvider<TSyntax> : LSPCompl
                 new[] { PermissionElementName, CrefAttributeName, $"{CrefAttributeName}=\"", "\"" },
                 new[] { SeeElementName, CrefAttributeName, $"{CrefAttributeName}=\"", "\"" },
                 new[] { SeeElementName, LangwordAttributeName, $"{LangwordAttributeName}=\"", "\"" },
+                new[] { SeeElementName, HrefAttributeName, $"{HrefAttributeName}=\"", "\"" },
                 new[] { SeeAlsoElementName, CrefAttributeName, $"{CrefAttributeName}=\"", "\"" },
+                new[] { SeeAlsoElementName, HrefAttributeName, $"{HrefAttributeName}=\"", "\"" },
```

Comment:
Unrelated: Should langword be offered in `<seealso/>`?

Reply:
no, `langword` is just for `see`